extension.js

```diff
@@ -104,6 +116,15 @@ class PipOnTop
       window._notifyPipTitleId = window.connect_after(
         'notify::title', this._checkTitle.bind(this));
     }
+    if (!window._windowPositionChangedId) {
+      window._windowPositionChangedId = window.connect_after(
+        'position-changed', this._onWindowChanged.bind(this));
+    }
+    if (!window._windowSizeChangedId) {
+      window._windowSizeChangedId = window.connect_after(
+        'size-changed', this._onWindowChanged.bind(this));
+    }
```

Comment:
This can potentially slow shell down by tracking multiple windows and having many new signals connected at once. Can't we just just connect these to PiP windows only after we determine that window `_isPipAble` maybe?